handler/src/main/java/io/netty/handler/ssl/Protocols.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+import io.netty.util.internal.ObjectUtil;
+
+import java.util.Arrays;
+
+/**
+ * TLS protocols
+ */
+public final class Protocols {
```

Comment:
Should we call this `SslProtocols` just to be more consistent ? Also you can then remove the duplicated definitions in `SslUtils` and make use of these here.

Reply:
@hyperxpro forgot to remove this class ?